lib/acts_as_taggable_on_mongoid/taggable/list_tags.rb

```diff
@@ -7,9 +7,15 @@ module ListTags
       extend ActiveSupport::Concern
 
       included do
-        class_attribute :tag_types
+        class_attribute :my_tag_types
```

Comment:
This was done due to inheritance issues.  If you access the `tag_types` for an inherited class, this makes a new set of values for the inherited class.